mesh/mesh.cpp

```diff
@@ -761,6 +761,10 @@ const GeometricFactors* Mesh::GetGeometricFactors(const IntegrationRule& ir,
       GeometricFactors *gf = geom_factors[i];
       if (gf->IntRule == &ir && (gf->computed_factors & flags) == flags)
       {
+         if (gf->RecomputeStatus())
```

Comment:
Do we want to add a `Get` or `Need`?

Reply:
Instead of ``->RecomputeStatus``? I like ``NeedRecompute`` we can change it to that if it is preferred.

Reply:
I prefer `NeedRecompute` too.